build.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
 	implementation "io.springfox:springfox-swagger-ui:3.0.0"
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
+	implementation 'org.springframework.boot:spring-boot-starter-validation'
```

Comment:
Isn't javax.validation:validation-api enough?

Reply:
Sadly, no. We get the following error:
```
Parameter 0 of constructor in com.iexec.blockchain.tool.ChainConfig required a bean of type 'javax.validation.Validator' that could not be found.
```

Reply:
I see thanks